mozci/task.py

```diff
@@ -346,7 +346,8 @@ def backfill(self, push):
                 "taskGroupId": decision_task.id,
                 "input": {
                     "times": 5
-                    if self.classification == "not classified"
+                    if self.classification
+                    in ["not classified", "new failure not classified"]
```

Comment:
Nit:
```suggestion
                    in ("not classified", "new failure not classified")
```
(tuple is immutable, so will consume slightly less memory)